modules/installation-special-config-raid-intel-vroc.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-special-config-raid-intel-vroc_{context}"]
+== Configuring an Intel(R) Virtual RAID on CPU (VROC) data volume
+
+Intel(R) VROC is a type of hybrid RAID, where some of the maintenance is offloaded to the hardware, but appears as software RAID to the operating system.
+
+The following procedure configures an Intel(R) VROC-enabled RAID1.
+
+.Procedure
+
+. Create the Intel(R) Matrix Storage Manager (IMSM) RAID container by running the following command:
++
+[source,terminal]
+----
+$ mdadm -CR /dev/md/imsm0 -e imsm -n2 /dev/nvme0n1 /dev/nvme1n1
+----
+
+. Create the RAID1 storage inside the container:
+
+.. Create a dummy RAID0 volume in front of the real RAID1 volume by running the following command:
++
+[source,terminal]
+----
+$ mdadm -CR /dev/md/dummy -l0 -n2 /dev/imsm0 -z10m --assume-clean
+----
+
+.. Create the real RAID1 array by running the following command:
++
+[source,terminal]
+----
+$ mdadm -CR /dev/md/coreos -l1 -n2 /dev/imsm0
+----
+
+.. Stop both RAID0 and RAID1 member arrays and delete the dummy RAID0 array with the following commands:
++
+[source,terminal]
+----
+$ mdadm -S /dev/md/dummy \
+  mdadm -S /dev/md/coreos \
+  mdadm --kill-subarray=0 /dev/md/imsm0
+----
+
+.. Restart the RAID1 arrays by running the following command:
++
+[source,terminal]
+----
+$ mdadm -A /dev/md/coreos /dev/md/imsm0
+----
+
+. Install {op-system} on the RAID1 device, and include the `rd.md.uuid` kernel argument by running the following command:
++
+[source,terminal]
+----
+$ coreos-installer install /dev/md/coreos \
+  --append-karg rd.md.uuid=<md_UUID> <1>
+----
+<1> The UUID of the IMSM container.
```

Comment:
We need to tell the user the where to get that UUID from (e.g. `mdadm --details --export /dev/md/imsm0`).